content/influxdb/v2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,45 @@ menu:
 weight: 101
 ---
 
+## v2.7.6 {date="2024-04-12"}
+
+### Bug Fixes
+
+- Fix `panic index out of range` error for invalid series keys.
+- Update broken Flux and performance tests.
+- Do not panic when querying empty tags.
+- Improve shard deletion.
+- Return and respect cursor errors.
+- Update constant time code to make password strength calculations more constant.
+
+### Features
+
+- Add optional [stricter password requirements](/influxdb/v2/reference/config-options/#strong-passwords).
+
+### Maintenance
+
+- Upgrade to Go 1.21.8.
+- Upgrade Go toolchain to 1.21.9.
```

Comment:
I think we should clarify here that the build is with 1.21.9.  I find the two contradictory statements (while true), confusing.